README.md

```diff
@@ -2,11 +2,9 @@
 
 This repository holds resources for an initiative to improve interoperability in building management, by focusing on standardising naming of connected devices.
 
-Being able to efficiently collect, analyse and leverage data insights from buildings is a catalyst for optimising building performance, improving the use of resources and moving towards predictive maintenance and buildings that can respond  to the climate emergency.
+Achieving a common standard across building devices’ naming is an important first step towards being able to efficiently collect, analyse and capture data insights from buildings, an thus optimise building performance and reduce the operational cost and environmental impact of managing buildings.  
```

Comment:
Hey Rita,

I would suggest changing to the following.....

Achieving a common standard across building devices’ naming is an important first step towards being able to efficiently collect, analyse and capture data insights from buildings, and thus optimise building performance to reduce the operational cost and environmental impact of managing buildings.

Reply:
Hey Richard,

Updated text as per suggestion.